src/isIsogram.test.js

```diff
@@ -6,4 +6,20 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`'true' if has no repeating letters`, () => {
+    expect(isIsogram('playgrounds')).toBe(true);
+  });
```

Comment:
- don't need to duplicate brackets 
- It's better to start description with `should be or `should return`
```suggestion
  it(`should return true if there are no repeating letters`, () => {
    expect(isIsogram('playgrounds')).toBe(true);
  });
```